src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -563,6 +565,22 @@ public function nodeName()
      * @throws \InvalidArgumentException When current node is empty
      */
     public function text(/* $default = null */)
+    {
+        if (\func_num_args() && ($default = func_get_arg(0))) {
```

Comment:
I don't get the code. It looks like that you still have `$default` AND `$normalizeWhitespace` overriding each other.

Reply:
I think we need two parameters. I don't see how we can pass both at this point

Reply:
huum, no, mea culpa, I think we need to check it's either a boolean or something else